scio.rb

```diff
@@ -0,0 +1,30 @@
+class Scio < Formula
+  desc "Scala API for Google Cloud Dataflow and Apache Beam"
+  homepage "https://github.com/spotify/scio/"
+  url "https://github.com/spotify/scio/releases/download/v0.2.0/scio-repl-0.2.0.jar"
+  sha256 "220e03dfd866d8475fc7943f60376ea4fe0862c3520674e46ef8262a09d1756c"
```

Comment:
According to `brew audit` they figure it out from the URL with ruby+regex magic.